app/code/Magento/Catalog/Console/Command/ImagesResizeCommand.php

```diff
@@ -71,7 +71,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if (!count($productIds)) {
             $output->writeln("<info>No product images to resize</info>");
             // we must have an exit code higher than zero to indicate something was wrong
```

Comment:
@okorshenko I'm wondering if the comment is correct in the first place. Should an error be triggered if there are no products? Running the command when there are no products / product images does no harm.

If exiting with an error is desired, we shoul also change the output from `<info>` to `<error>`.

Reply:
Hi @nfourteen , I agree with @mzeis , it's ok if the status code will be 'success' if no products in the store. Regarding the comment in the line above - status code was changed from 'failure' to 'success' in scope of bugfix and developer forget to remove this line. For now, it would be better remove comment line and leave status code as is.